FlowerShop/FlowerShop/Scenes/OrderDetails/OrderDetailsService.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  OrderDetailsService.swift
+//  FlowerShop
+//
+//  Created by Andrei Oltean on 23/02/2020.
+//  Copyright © 2020 Andrei Oltean. All rights reserved.
+//
+
+import UIKit
+
+enum ImageResponse {
+    case success(UIImage)
+    case failure(String)
+}
+
+class OrderDetailsService {
+    let network = AlamofireNetwork.shared
```

Comment:
What do you think about using dependency injection for this object here, instead of creating it inside the OrderDetailsService class?